src/dotnet/commands/dotnet-restore/LocalizableStrings.cs

```diff
@@ -14,6 +14,10 @@ internal class LocalizableStrings
 
         public const string CmdSourceOptionDescription = "Specifies a NuGet package source to use during the restore.";
 
+        public const string CmdRuntimeOption = "RUNTIME_IDENTIFIER";
```

Comment:
@srivatsn Would it be possible to get these localized along with the localization for dotnet add package?

if so, please approve this for merging and I will merge it.